docs/install/desktop/README.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Desktop
+---
+
+# IPFS desktop application
```

Comment:
similarly, "IPFS Desktop" is a name, not closely associated with "Desktop" as an amorphous category.

Reply:
6784034 fixes this issue.